dependencies.gradle

```diff
@@ -52,7 +53,8 @@ def VERSIONS = [
         'org.hdrhistogram:HdrHistogram:latest.release',
         'org.hibernate:hibernate-entitymanager:latest.release',
         'org.hsqldb:hsqldb:latest.release',
-        'org.jooq:jooq:latest.release',
+        // Pin version temporarily to restore builds. See gh-1852
```

Comment:
Included so the build will pass, can be removed once https://github.com/micrometer-metrics/micrometer/issues/1852 is resolved